app/src/main/java/de/westnordost/streetcomplete/quests/QuestModule.kt

```diff
@@ -54,6 +54,7 @@ import de.westnordost.streetcomplete.quests.oneway_suspects.data.TrafficFlowSegm
 import de.westnordost.streetcomplete.quests.oneway_suspects.data.WayTrafficFlowDao
 import de.westnordost.streetcomplete.quests.opening_hours.AddOpeningHours
 import de.westnordost.streetcomplete.quests.atm_operator.AddAtmOperator
+import de.westnordost.streetcomplete.quests.bike_parking_type.AddBarrierType
```

Comment:
```suggestion
import de.westnordost.streetcomplete.quests.barrier_type.AddBarrierType
```

Reply:
Thanks! I already commited it (not pushed), together with one more reference.